app/components/primer/avatar_stack_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <%= render body_component do %>
+    <% avatars.each_with_index do |avatar, i| %>
+      <% if i == 2 %>
```

Comment:
Do we want to make this value configurable?

Reply:
I don't think Primer supports other values:

![image](https://user-images.githubusercontent.com/11280312/108260511-06b3a880-7128-11eb-84c7-40e8b4dbb6ee.png)